src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -200,6 +200,19 @@ public Command zero() {
         });
   }
 
+  public void seedEncoder() {
```

Comment:
The front absolute encoder needs to be inverted (1.0 - climberFrontEncoder.get()). You might consider using a helper function.
Also need to add the kClimberOffsetAngle.


Look at the zero() method above this one. Once this method is fixed, zero() should call it as a runOnce Command instead.